src/maps/london/production.ts

```diff
@@ -0,0 +1,11 @@
+import { PhaseEngine } from "../../engine/game/phase";
+import { Phase } from "../../engine/state/phase";
+
+export class LondonPhaseEngine extends PhaseEngine {
+  phaseOrder(): Phase[] {
+    // No goods growth phase in London
+    const previous = super.phaseOrder();
```

Comment:
The remove function does precisely this, so you should be able to just call: `return remove(super.phaseOrder(), Phase.GOODS_GROWTH);`